drud/ddev
# Pull images earlier to avoid confusion

## The Problem/Issue/Bug:

* In testing for v1.18.0, it was discovered that you can have a long, inexplicable pause on `ddev start` after it asks if you want to poweroff. It's the poweroff inexplicably silently pulling the new webimage. So put the pull into start before the option to poweroff

## How this PR Solves The Problem:

Use the more vocal pull option (dockerutils.Pull) instead of the library client to pull. This will fix that issue for RunSimpleContainer() generally.

## Manual Testing Instructions:

* Manually change the last_used_version in ~/.ddev/config.yaml to v1.17.7
* Manually delete the webimage in `ddev version`, `docker rmi drud/ddev-webserver...`
* `ddev start`
* It should pull the images, then ask for permission to poweroff.




<a href="https://gitpod.io/#https://github.com/drud/ddev/pull/3257"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
